src/test/java/org/apache/commons/io/input/BoundedInputStreamTest.java

```diff
@@ -203,10 +268,10 @@ void testCounts(final long startCount) throws Exception {
 
     @Test
     void testMarkReset() throws Exception {
-        final byte[] helloWorld = "Hello World".getBytes(StandardCharsets.UTF_8);
+        final byte[] helloWorld = "Hello World".getBytes(UTF_8);
```

Comment:
In general, I would not change the access style in a PR because there is now more to review.